src/utils/gitHubUtils.ts

```diff
@@ -103,9 +103,20 @@ export async function getGitHubQuickPicksWithLoadMore<TResult, TParams>(
     }
 }
 
-export async function getGitHubAccessToken(): Promise<string> {
+export async function getGitHubAccessToken(context?: IActionContext): Promise<string> {
```

Comment:
Why is context optional?

Reply:
Because it's kind of a pain to make it required since it's used in a lot of code, some of which does not context.  But most of those are one-off calls (like getting the Actions).  In the case the context _is_ passed in is during the create static web app wizard, where I thought we'd be more interested in the cancel step telemetry.

Reply:
Will be handled by _someone_ _after_ release